cli/ballerina-cli/src/main/java/io/ballerina/cli/task/DumpBuildTimeTask.java

```diff
@@ -79,6 +81,19 @@ private Path getBuildTimeFilePath(Project project) {
         if (project.kind().equals(ProjectKind.BUILD_PROJECT)) {
             return project.sourceRoot().resolve("target").resolve(BUILD_TIME_JSON).toAbsolutePath();
         }
-        return currentDir.resolve("build-time.json").toAbsolutePath();
+        return currentDir.resolve(BUILD_TIME_JSON).toAbsolutePath();
+    }
+
+    private void printBuildTime(BuildTime buildTime) {
+        this.out.println("\ttimestamp : " + buildTime.timestamp);
```

Comment:
Shall we avoid hardcoding the properties? You can use `gson.fromJson` to convert it to a `JsonObject`  and then iterate the map of entries

Reply:
E.g. `JsonObject obj = new Gson().fromJson(BuildTime.getInstance(), JsonObject.class)`

Reply:
Added